src/styles/blocks/rating.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @include flex($direction: row, $items: center);
```

Comment:
Ensure that the `flex` mixin is defined and imported correctly. This mixin is used to apply flexbox properties, and any missing definition could lead to styling issues.